melodic/distribution.yaml

```diff
@@ -3442,10 +3442,11 @@ repositories:
       url: https://github.com/ros-drivers/microstrain_3dmgx2_imu.git
       version: indigo-devel
   microstrain_mips:
-    doc:
```

Comment:
Did you mean to remove the doc entry?  It's not required, but generally recommended.

Reply:
I did not! Thanks!